classes/webservice/XmlWebService.inc.php

```diff
@@ -63,7 +63,9 @@ function setReturnType($returnType) {
 	function &call(&$webServiceRequest) {
 		// Call the web service
 		$xmlResult = parent::call($webServiceRequest);
-
+		if (Config::getVar('debug', 'log_web_service_info')) {
+			error_log('Time: ' . date('c') . "\nRequest: " . print_r($webServiceRequest, true) . "\nResponse: " . print_r($xmlResult, true) . "\nLast response status: " . $this->_lastResponseStatus . "\n", 0);
```

Comment:
Mind also removing the `0` magic number from the call to `error_log`? It's implicit and I hate magic numbers.